crates/services/producer/src/block_producer.rs

```diff
@@ -387,11 +392,11 @@ where
                 best: highest,
                 previous_block: previous_da_height,
             }
-            .into());
+            .into())
         }
 
         if highest == previous_da_height {
-            return Ok(highest);
+            return Ok(highest)
```

Comment:
It would be nice to avoid unnecessary changes

Reply:
Done, still trying to figure out the formatter configuration to avoid these. See 5b4829574c5d9214adbd5d858a5dcd3371ae5c30